### ISSUE TITLE
Feature Request: workflow templates that chain agents together

### ISSUE THREAD

**Issue by nemosmithasf** [huginn/huginn#2769]
It's a pretty intuitive request actually. Consider this pseudo-workflow:

```
// Check website for change
var data = Webagent("www.url.com")
var hasChanged = comparisonAgent(data, oldData)
emailAgent(data + oldData, "person@gmail.com")
```

If we wanted to duplicate this workflow, we'd have to clone 3 agents and relink them. It would be useful if we could just make it a workflow template/function:

```
function trackWebsite(url){
    var data = Webagent(url)
    var hasChanged = comparisonAgent(data, oldData)
    emailAgent(data + oldData, "person@gmail.com")
}
```

Then it's just a matter of deploying the template. This makes complex workflows must easier to use.

**Comment by dsander** [comment]
I agree that this sounds useful. You can kind of trick Huginn into doing this by exporting a Scenario as JSON and then modify the root `guid` to something different. Huginn will then let you import the JSON as a new Scenario.
